rplugin/python3/denite/source/man.py

```diff
@@ -36,3 +48,18 @@ def gather_candidates(self, context):
         candidates = [{'word': result, 'addr': result, 'kind': 'man'} for result in candidates]
 
         return candidates
+
+    @staticmethod
+    def _get_arg( context, idx, default ):
```

Comment:
Please remove the whitespace around the parentheses.

Reply:
Fixed